MRETestBed/Assets/TestBed Assets/Scripts/UserInfo.cs

```diff
@@ -28,8 +28,32 @@ public UserInfo(Guid id)
     public event MWEventHandler BeforeAvatarDestroyed;
     public event MWEventHandler AfterAvatarCreated;
 
+    private static Transform FindChildRecursive(Transform parent, string name)
+    {
```

Comment:
This is a useful helper function and should be in an appropriate file in the Util folder and namespace.  Not private in this class I would think.

Reply:
I don't know how often this method would actually be used, as we're generally not searching for Transforms, but I agree that Utils is the right place for this.

---
In reply to: [261803456](https://github.com/Microsoft/mixed-reality-extension-unity/pull/42#discussion_r261803456) [](ancestors = 261803456)

Reply:
No need to move this to the UnityRuntime project unless we have a need for it there.

Reply:
I like the one-liner though :)